.github/CODEOWNERS

```diff
@@ -711,6 +711,12 @@
 # ServiceLabel: %Network - Network Virtual Appliance
 # ServiceOwners:                                                   @nvasuppgithub
 
+# PRLabel: %Network - Mobile
```

Comment:
I missed that these were not moved into the management section as requested.   Please move as mentioned in my other comment.

Reply:
Ah oops - I had moved them but looks like I moved them too far back when doing a markup.

I've put it back in the right place (and changed it back to Network - Private Mobile Network, as I think Network - Mobile is less discoverable for the team)

Reply:
The label needs to stay `Network - Mobile` as that is centrally managed and synchronized across repositories.